### PR TITLE
Sim

### PR DESCRIPTION
**Pull request by NGovani**
Initial simulation commit, sets the format for how metrics should be presented in future analysis

### CODE REVIEW

**Comment by ezgimez** [comment]
can @yp717  check this PR plz?

if @yp717  is unavailable, I can check this PR _belatedly_ cuz I'm in deep IIGO stuff now

File: 15_simulations/index.tex [hunk 1]
```diff
@@ -1 +1,93 @@
 \chapter{Simulations}
+\section{Introduction}
+\label{sec:Simulations:Intro}
+
+The purpose of performing simulations and analysing their results is to explore the efficacy of each agent's strategy under different conditions. We would like to answer questions such as: does the benefit of organisation outweigh the cost? Or, can the agent strategies overcome the foraging dilemma outlined in Section INSERT REF LATER? The analysis encapsulate the purpose of the entire project, and is the final result of all the work we have put in. In this chapter we aim to prove that our platform and the agents are capable of display interesting behaviour and that we can draw conclusions about how behaviors certain behaviours and environmental factors affect a multi-agent system.
```
Comment (lhl2617) [inline 15_simulations/index.tex]: analyses encapsulate or analysis encapsulates

Comment (lhl2617) [reply 15_simulations/index.tex]: all the work ~we have put in~ done

Comment (lhl2617) [reply 15_simulations/index.tex]: ~display~displaying

Comment (lhl2617) [reply 15_simulations/index.tex]: how ~behaviors~?

File: 15_simulations/index.tex [hunk 2]
```diff
@@ -1 +1,93 @@
...
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics, these numerical values or visualisation give us information on the events of the game that we can then use to compare numerous simulation and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
+
+
+\begin{itemize}
+    \item \textbf{Archipelago Survivability}: Turns until all the islands in the archipelago die.
+    \item \textbf{First Island Death}: Turns until a single island dies.
+    \item \textbf{Island Resources$^1$$^2$}: Resource an island has each turn
```
Comment (lhl2617) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Island Resources$^1$$^2$}: Resource an island has each turn.
```

File: 15_simulations/index.tex [hunk 3]
```diff
@@ -1 +1,93 @@
...
+
+The purpose of performing simulations and analysing their results is to explore the efficacy of each agent's strategy under different conditions. We would like to answer questions such as: does the benefit of organisation outweigh the cost? Or, can the agent strategies overcome the foraging dilemma outlined in Section INSERT REF LATER? The analysis encapsulate the purpose of the entire project, and is the final result of all the work we have put in. In this chapter we aim to prove that our platform and the agents are capable of display interesting behaviour and that we can draw conclusions about how behaviors certain behaviours and environmental factors affect a multi-agent system.
+
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics, these numerical values or visualisation give us information on the events of the game that we can then use to compare numerous simulation and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs.
```
Comment (lhl2617) [inline 15_simulations/index.tex]:
```suggestion
To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics: these numerical values give us information on the events of the game that we can then use to compare numerous simulations and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
```

File: 15_simulations/index.tex [hunk 4]
```diff
@@ -1 +1,93 @@
...
+
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics, these numerical values or visualisation give us information on the events of the game that we can then use to compare numerous simulation and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
+
+
+\begin{itemize}
+    \item \textbf{Archipelago Survivability}: Turns until all the islands in the archipelago die.
+    \item \textbf{First Island Death}: Turns until a single island dies.
```
Comment (lhl2617) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{First Island Death}: Turns until any island dies.
```

(two islands can die in the same turn)

File: 15_simulations/index.tex [hunk 5]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{IIGO Tax$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for tax and amount the island has paid.
+    \item \textbf{IIGO Sanctions$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for sanctions and amount the island has paid.
+    
+\end{itemize}
+\small{$^1$ Presented on a per island basis. $^2$ Presented as a graph}
+
+\section{Baseline Simulation}
+\label{sec:Simulations:baseline}
+
+In order to make any meaningful analysis one must first have baseline from which to compare. In our case, this baseline is a 'normal' set of environment conditions which allow the agents to thrive for a significant period of time, around 100 turns. From this baseline we can then start adjusting the configuration of the simulation, either making disaster more frequent/deadly, or making resources more sparse.
```
Comment (lhl2617) [inline 15_simulations/index.tex]:
```suggestion
In order to make any meaningful analysis one must first have a baseline from which to compare. In our case, this baseline is a \emph{normal} set of environment conditions which allows the agents to thrive for a significant period of time, chosen at around 100 turns. From this baseline we can then start adjusting the configuration of the simulation, either making disasters more frequent/deadly, or making resources more sparse.
```

**Comment by ezgimez** [comment]
let me startt... shall we

File: 15_simulations/index.tex [hunk 6]
```diff
@@ -1 +1,93 @@
...
+
+
+\subsection{Baseline IIGO Tax, Allocations and Sanctions}
+\label{subsec:Simulations:baseline:IIGO}
+
+
+\subsection{Baseline IIGO Roles}
+\label{subsec:Simulations:baseline:IIGO_roles}
+
+\subsection{Baseline Trading}
```
Comment (ezgimez) [inline 15_simulations/index.tex]: trading?? do we have trading? isn't it gifting or smth

File: 15_simulations/index.tex [hunk 7]
```diff
@@ -1 +1,93 @@
...
+    \textbf{Disasters Survived}         &       \\
+    \textbf{ADDM}                       &       \\
+    \textbf{AFS}                        &       \\ \hline
+\end{tabular}
+\caption{Numerical Metrics}
+\end{table}
+
+\subsection{Baseline IFS}
+\label{subsec:Simulations:baseline:ifs}
+\begin{table}[htb]
```
Comment (ezgimez) [inline 15_simulations/index.tex:66]: hm.. the widths of the columns are not ideal.
can you enlarge the width of the very first column xxx

File: 15_simulations/index.tex [hunk 8]
```diff
@@ -1 +1,93 @@
...
+Note: If a metric is not of interest to a certain area of exploration it will not be shown.
+
+\subsection{Baseline Numeric Metrics}
+\label{subsec:Simulations:baseline:num_metrics}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{|l|l|}
+    \hline
+    \textbf{Metric}                     & \textbf{Value} \\ \hline
+    \textbf{Archipelago Survivability}  &       \\
```
Comment (ezgimez) [inline 15_simulations/index.tex]: ok, let's be conssitent..
I think you should also include LINES in this table.... 
just like the table you have below :)

File: 15_simulations/index.tex [hunk 9]
```diff
@@ -1 +1,93 @@
...
+
+\section{Baseline Simulation}
+\label{sec:Simulations:baseline}
+
+In order to make any meaningful analysis one must first have a baseline from which to compare. In our case, this baseline is a \emph{normal} set of environment conditions which allows the agents to thrive for a significant period of time, chosen at around 100 turns. From this baseline we can then start adjusting the configuration of the simulation, either making disasters more frequent/deadly, or making resources more sparse.
+
+Note: If a metric is not of interest to a certain area of exploration it will not be shown.
+
+\subsection{Baseline Numeric Metrics}
+\label{subsec:Simulations:baseline:num_metrics}
```
Comment (ezgimez) [inline 15_simulations/index.tex:48]: oh god, good labellll

File: 15_simulations/index.tex [hunk 10]
```diff
@@ -1 +1,93 @@
 \chapter{Simulations}
+\section{Introduction}
+\label{sec:Simulations:Intro}
+
+The purpose of performing simulations and analysing their results is to explore the efficacy of each agent's strategy under different conditions. We would like to answer questions such as: does the benefit of organisation outweigh the cost? Or, can the agent strategies overcome the foraging dilemma outlined in Section INSERT REF LATER? The analysis encapsulates the purpose of the entire project, and is the final result of all the work done. In this chapter we aim to prove that our platform and the agents are capable of displaying interesting behaviour and that we can draw conclusions about how certain behaviours and environmental factors affect a multi-agent system.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
The purpose of performing simulations and analysing their results is to explore the efficacy of each agent's strategy under different conditions. We would like to answer questions such as: 

\begin{itemize}
\item Does the benefit of running the IIGO outweigh its cost? 
\item Can the agent strategies overcome the foraging dilemma outlined in Section INSERT REF LATER? 
\end{itemize}
 ```

Comment (ezgimez) [reply 15_simulations/index.tex]: could you include more questions PLZZ

Comment (ezgimez) [reply 15_simulations/index.tex]:
```suggestion
The analysis encapsulates the purpose of the entire coursework and therefore, is the final result of all the work done. In this chapter, we aim to prove that our platform and the agents are capable of displaying interesting behaviour and hence, we can draw conclusions about how certain behaviours and environmental factors affect a multi-agent system.
```

I liked the last sentence -- good analysis

Comment (ezgimez) [reply 15_simulations/index.tex]: well ok GitHub didn't understand my intention, but it must be clear

File: 15_simulations/index.tex [hunk 11]
```diff
@@ -1 +1,93 @@
...
+
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics: these numerical values give us information on the events of the game that we can then use to compare numerous simulations and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
To assist in this analysis, we need to draw quantifiable results from the simulations. Thus enters the metrics: These numerical values give us information on the events of the game that we can then use to compare numerous simulations and draw insights into what influences the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
```

File: 15_simulations/index.tex [hunk 12]
```diff
@@ -1 +1,93 @@
...
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics: these numerical values give us information on the events of the game that we can then use to compare numerous simulations and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
+
+
+\begin{itemize}
+    \item \textbf{Archipelago Survivability}: Turns until all the islands in the archipelago die.
+    \item \textbf{First Island Death}: Turns until any island dies.
+    \item \textbf{Island Resources$^1$$^2$}: Resource an island has each turn.
```
Comment (ezgimez) [inline 15_simulations/index.tex]: what is this ^1 ??

Comment (ezgimez) [reply 15_simulations/index.tex]: lol, do you wanna make a footnote or smth :D

File: 15_simulations/index.tex [hunk 13]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Gini Index}: A measure of how fair the distribution of resources is across the archipelago.
+    \item \textbf{Disasters Survived}: Number of disasters the archipelago has survived.
+    \item \textbf{Island Trading$^1$$^2$}: Resources an island has gifted to other islands.
+    \item \textbf{Average Disaster Damage Mitigated(ADDM)}: Average disaster damage mitigated by the common pool.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Average Disaster Damage Mitigated (ADDM)}: Average disaster damage mitigated by the common pool.
```

File: 15_simulations/index.tex [hunk 14]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Island Foraging Statistics(IFS)$^1$}: Resources an island has invested and gained from foraging.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Island Foraging Statistics (IFS)$^1$}: Amount of resources an island has invested and gained from foraging.
```

File: 15_simulations/index.tex [hunk 15]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Archipelago Foraging Sustainability(AFS)}: The average net forage returns across all islands.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Archipelago Foraging Sustainability (AFS)}: The average net forage returns across all islands.
```

File: 15_simulations/index.tex [hunk 16]
```diff
@@ -1 +1,93 @@
...
+
+\section{Metrics}
+\label{sec:Simulations:Metric}
+
+To assist in this analysis we need to draw quantifiable results from the simulations. Thus enters the metrics: these numerical values give us information on the events of the game that we can then use to compare numerous simulations and draw insights into what effect the changing of certain parameters has on the game. Some metrics are also worth knowing on a turn by turn basis and as such have been represented as graphs. 
+
+
+\begin{itemize}
+    \item \textbf{Archipelago Survivability}: Turns until all the islands in the archipelago die.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Archipelago Survivability}: Number of turns until all the islands in the archipelago die.
```

File: 15_simulations/index.tex [hunk 17]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{First Island Death}: Turns until any island dies.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{First Island Death}: Number of turns until any island dies.
```

File: 15_simulations/index.tex (hunk 12 above)
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Island Resources$^1$$^2$}: Amount of resources an island has each turn.
```

File: 15_simulations/index.tex [hunk 18]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Island Resources$^1$$^2$}: Resource an island has each turn.
+    \item \textbf{Gini Index}: A measure of how fair the distribution of resources is across the archipelago.
```
Comment (ezgimez) [inline 15_simulations/index.tex:27]: good wording <3

File: 15_simulations/index.tex [hunk 19]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Disasters Survived}: Number of disasters the archipelago has survived.
```
Comment (ezgimez) [inline 15_simulations/index.tex]: survived... meaning?
everyone survived or at least 1 island survived??
revise

File: 15_simulations/index.tex (hunk 13 above)
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{Average Disaster Damage Mitigated(ADDM)}: Average amount of disaster damage mitigated by the common pool.
```

File: 15_simulations/index.tex [hunk 20]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Island Trading$^1$$^2$}: Resources an island has gifted to other islands.
+    \item \textbf{Average Disaster Damage Mitigated(ADDM)}: Average disaster damage mitigated by the common pool.
+    \item \textbf{Island Foraging Statistics(IFS)$^1$}: Resources an island has invested and gained from foraging.
+    \item \textbf{Archipelago Foraging Sustainability(AFS)}: The average net forage returns across all islands.
+    \item \textbf{IIGO Roles$^2$}: The power each island has at any turn in the game. Power in this case is occupying one of the three roles in IIGO.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{IIGO Roles$^2$}: The power each island has at any turn in the game\footnote{Power, in this case, refers to occupying one of the three roles in IIGO}.
```

or smth like that ..  fix my english plz

File: 15_simulations/index.tex [hunk 21]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{IIGO Allocations$^1$$^2$}: Amount of resources allocated to each island by the president and the amount the island has taken.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{IIGO Allocations$^1$$^2$}: Amount of resources allocated to each island by the President and the amount the island has taken from the common pool.
```

File: 15_simulations/index.tex [hunk 22]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{IIGO Tax$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for tax and amount the island has paid.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{IIGO Tax$^1$$^2$}: Amount of resources an island is expected to contribute to the common pool in terms of tax and amount of tax an island has actually paid.
```

File: 15_simulations/index.tex [hunk 23]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{IIGO Sanctions$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for sanctions and amount the island has paid.
+    
+\end{itemize}
+\small{$^1$ Presented on a per island basis. $^2$ Presented as a graph}
```
Comment (ezgimez) [inline 15_simulations/index.tex]: footnote plz

File: 15_simulations/index.tex [hunk 24]
```diff
@@ -1 +1,93 @@
...
+    \item \textbf{Disasters Survived}: Number of disasters the archipelago has survived.
+    \item \textbf{Island Trading$^1$$^2$}: Resources an island has gifted to other islands.
+    \item \textbf{Average Disaster Damage Mitigated(ADDM)}: Average disaster damage mitigated by the common pool.
+    \item \textbf{Island Foraging Statistics(IFS)$^1$}: Resources an island has invested and gained from foraging.
+    \item \textbf{Archipelago Foraging Sustainability(AFS)}: The average net forage returns across all islands.
+    \item \textbf{IIGO Roles$^2$}: The power each island has at any turn in the game. Power in this case is occupying one of the three roles in IIGO.
+    \item \textbf{IIGO Allocations$^1$$^2$}: Amount of resources allocated to each island by the president and the amount the island has taken.
+    \item \textbf{IIGO Tax$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for tax and amount the island has paid.
+    \item \textbf{IIGO Sanctions$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for sanctions and amount the island has paid.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
    \item \textbf{IIGO Sanctions$^1$$^2$}: Amount of resources an island is expected to contribute to the common pool due to the imposed sanctions and amount of contribution the island has actually did for the imposed sanctions.
```

File: 15_simulations/index.tex (hunk 23 above)
Comment (ezgimez) [reply 15_simulations/index.tex]: presented as a graph. 

full-stop svp

File: 15_simulations/index.tex [hunk 25]
```diff
@@ -1 +1,93 @@
...
+    
+\end{itemize}
+\small{$^1$ Presented on a per island basis. $^2$ Presented as a graph}
+
+\section{Baseline Simulation}
+\label{sec:Simulations:baseline}
+
+In order to make any meaningful analysis one must first have a baseline from which to compare. In our case, this baseline is a \emph{normal} set of environment conditions which allows the agents to thrive for a significant period of time, chosen at around 100 turns. From this baseline we can then start adjusting the configuration of the simulation, either making disasters more frequent/deadly, or making resources more sparse.
```
Comment (ezgimez) [inline 15_simulations/index.tex]:
```suggestion
In order to make any meaningful analysis, one must first have a baseline against which to compare other performances. In our case, this baseline is a \emph{normal} set of environmental conditions which allows the agents to thrive for a significant period of time, chosen to be at around 100 turns. From this baseline, we can then start adjusting the configuration of the simulation by either making disasters more frequent and deadly or making resources more sparse.
```

**Review by ezgimez — CHANGES_REQUESTED** [review]

File: 15_simulations/index.tex (hunk 8 above)
Comment (ezgimez) [reply 15_simulations/index.tex]: well ok LOL

**Comment by ezgimez** [comment]
lgtm

although I don't like the inconsistency in table styles, due to time constraints,  _I don't care_ about this at the moment

**Review by ezgimez — APPROVED** [review]

**Pull request merged**